sdk/communication/communication-chat/samples/javascript/README.md

```diff
@@ -0,0 +1,40 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+urlFragment: communication-chat-javascript
+---
+
+## Prerequisites
+
+The sample is compatible with Node.js >= 8.0.0.
+
+You need [an Azure subscription][freesub] and [an Azure Communication Service Instance][azcomsvc] to run these sample program.
+
+Adapting the samples to run in the browser may require some additional consideration. For details, please see the [package README][package].
+
+## Setup
+
+To run the sample using the published version of the package:
+
+1. Install the dependencies using `npm`:
+
+```bash
+npm install
+```
+
+2. Edit the file `sample.env`, adding the correct credentials to access the Azure service and run the samples. Then rename the file from `sample.env` to just `.env`. The sample programs will read this file automatically.
+
+3. Run whichever samples you like:
+
+```bash
+node threadOperationSample.js
```

Comment:
Can we rename this file? Everything in this folder is a sample, maybe just `createThread.ts`.

Ideally we would split this one sample up in multiple for each basic scenario, but we can do that later.